include/utils/json.hpp

```diff
@@ -170,6 +177,19 @@ std::vector<T> GetArrayValue(const CaseInsensitiveObjectWrapper& object, const s
     return result;
 }
 
+/**
+ * Gets value array by key.
+ *
+ * @param object json object.
+ * @param key key.
+ * @return std::vector<T>.
+ */
+template <typename T>
+std::vector<T> ParseValueArray(const CaseInsensitiveObjectWrapper& object, const std::string& key)
```

Comment:
What is the difference between `ParseValueArray` and `GetArrayValue`? According to the description they do similar things but name is very different.